package.json

```diff
@@ -21,14 +21,14 @@
   "scss": "./dist/angular-material.scss",
   "devDependencies": {
     "add-stream": "^1.0.0",
-    "angular": "^1.5.0",
-    "angular-animate": "^1.5.0",
-    "angular-aria": "^1.5.0",
-    "angular-messages": "^1.5.0",
-    "angular-mocks": "^1.5.0",
-    "angular-route": "^1.5.0",
-    "angular-sanitize": "^1.5.0",
-    "angular-touch": "^1.5.0",
+    "angular": ">=1.5.0 <1.7.0",
```

Comment:
For versions 1.1.x, we do not want to exclude AngularJS 1.3.x, 1.4.x. 
I think will need something like ">=1.3.17 <1.7.0".

> This will need testing of course.